app/clients/aws/s3bucket.py

```diff
@@ -65,4 +78,50 @@ def get_s3_url(region: str, bucket: str, key: str) -> str:
     return f"https://{bucket}.s3.{region}.amazonaws.com/{key}"
 
 
+def upload_json_to_s3(
+    s3_client: AWSClient, config: S3UploadConfig, json_data: dict[str, Any]
+) -> None:
+    """
+    Upload a JSON file to S3
+
+    :param S3UploadConfig config: The configuration required for the upload.
+    :param dict[str, Any] json_data: The json data to be uploaded to S3.
+    :raises Exception: on any error when uploading the file to S3.
+    """
+    try:
+        s3_client.put_object(
+            Bucket=config.bucket_name,
+            Key=config.object_name,
+            Body=json.dumps(json_data),
+            ContentType="application/json",
+        )
+        _LOGGER.info(
+            f"🎉 Successfully uploaded JSON to S3: {config.bucket_name}/{config.object_name}"
+        )
+    except Exception as e:
+        _LOGGER.error(f"💥 Failed to upload JSON to S3:{e}]")
+        raise
+
+
+def upload_ingest_json_to_s3(
+    ingest_id: str, corpus_import_id: str, data: dict[str, Any]
+) -> None:
+    """
+    Upload an ingest JSON file to S3
+
+    :param str ingest_id: The uuid of the ingest action.
+    :param str corpus_import_id: The import_id of the corpus the ingest data belongs to.
+    :param dict[str, Any] json_data: The ingest json data to be uploaded to S3.
+    """
+    ingest_upload_bucket = os.environ["INGEST_JSON_BUCKET"]
```

Comment:
Is there a reason this is not part of config like the  bucket in the other upload function?